DEVCONTAINER.md

```diff
@@ -0,0 +1,141 @@
+# Using Development Containers for development and testing
+
+You can set up a development environment for this project using [Development Containers](https://containers.dev/), aka. devcontainers.
```

Comment:
Since this is an acronym rather than an abbreviation, either use "a.k.a." or no punctuation at all :)

```suggestion
You can set up a development environment for this project using [Development Containers](https://containers.dev/), aka devcontainers.
```